orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/config/ManifestEvaluatorConfigurationProperties.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.tasks.manifest.config;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
i think the preferred route is to not make this class a component, but instead, mark the consumer (`ManifestEvaluator`) with `@EnableConfigurationProperties`

Reply:
@marchello2000 Sure - we can do it that way. I looked at most of the way CP's are done in Orca today and saw both patterns and went with the former. From what I have seen there isn't any real benefit from doing it one way or the other. Do you know why @EnableConfigurationProperties with a POJO is the preferred?